bridge-impl/src/main/java/com/liferay/faces/bridge/context/internal/ExternalContextImpl.java

```diff
@@ -825,6 +825,7 @@ else if (portletPhase == Bridge.PortletPhase.RENDER_PHASE) {
 
 					try {
 						BridgeURL bridgeRedirectURL = bridgeURLFactory.getBridgeRedirectURL(facesContext, url, null);
+						bridgeRedirectURL.removeParameter(BridgeExt.REDIRECT_PARAMETER);
```

Comment:
This fixes the problem where the (non-namespaced) `_jsfBridgeRedirect=true` parameter appeared after an Ajax redirect.